Makefile

```diff
@@ -0,0 +1,53 @@
+WORKDIR := /tsfresh
+TEST_IMAGE := tsfresh-test-image
+TEST_DOCKERFILE := Dockerfile.testing
+TEST_CONTAINER := tsfresh-test-container
+# >= 3.9.2 ---> https://github.com/dask/distributed/issues/7956
+PYTHON_VERSIONS := "3.7.12 3.8.12 3.9.12 3.10.12 3.11.0"
+BLACK_VERSION := 22.12.0
+# Isort 5.12.0 not supported with python 3.7.12
+ISORT_VERSION := 5.12.0
+
+build-testenv:
+	docker build \
+			-f $(TEST_DOCKERFILE) \
+			-t $(TEST_IMAGE) \
+			--build-arg PYTHON_VERSIONS=$(PYTHON_VERSIONS) \
+			.
+
+# Tests `PYTHON_VERSIONS`, provided they are also
+# specified in setup.cfg `envlist`
+test-all-testenv: clean build-testenv
+	docker run --rm \
+			--name $(TEST_CONTAINER) \
+			-v .:$(WORKDIR) \
```

Comment:
Does I maybe make sense to mount the folder read-only?

Reply:
Does not look like read only plays nicely

```bash
docker: Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/var/lib/docker/volumes/egg_artifacts/_data" to rootfs at "/tsfresh/tsfresh.egg-info": mkdir /var/lib/docker/overlay2/f1e44a21b8f2b9407dbce127e0383601fcc372b2dda05cae82e9c2a6de33df42/merged/tsfresh/tsfresh.egg-info: read-only file system: unknown.
```